src/components/header/navbar/Topnavbar.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from "react";
+import {
+  Pane,
+  TabNavigation,
+  Tab,
+  SearchInput,
+  IconButton,
+  ShoppingCartIcon,
+  StarIcon,
+  Tooltip,
+  majorScale,
+} from "evergreen-ui";
+import nike from "../../../assets/img/nike.png";
+
+class Topnavbar extends Component {
+  //state declaration
+  constructor(props) {
+    super(props);
+    this.state = {
+      catagories: [
+        {
+          id: 1,
+          title: "NEW RELEASES",
+          hover: false,
+        },
+        {
+          id: 2,
+          title: "MEN",
+          hover: false,
+        },
+        {
+          id: 3,
+          title: "WOMEN",
+          hover: false,
+        },
+        {
+          id: 4,
+          title: "KIDS",
+          hover: false,
+        },
+        {
+          id: 5,
+          title: "CUSTOMIZE",
+          hover: false,
+        },
+        {
+          id: 6,
+          title: "SALE",
+          hover: false,
+        },
+        {
+          id: 7,
+          title: "COLLECTION",
+          hover: false,
+        },
+      ],
+    };
+  }
+
+  tabStyle = (hover) => {
+    return {
+      
+      padding: "10px",
+      height: "71px",
+      backgroundColor: "#fff",
+      border: "0px",
+      borderRadius: "0px",
+      borderBottomWidth: 1,
+      borderBottomStyle: "solid",
+      borderBottomColor: hover ? "#000000" : "#ffffff",
+      color: "#000",
+      textAlign: "center",
+      fontSize: "15px",
+    };
+  };
```

Comment:
Shouldn't this be in a `.css` file? If you do this part in `css`, I think the `hover` could be done more easily.

Reply:
as we are using react so hover will not be done with css, so for that reason i think this needs to be here in this file, although i'm not completely sure. @RasheeqZaman

Reply:
must do!